### DISCUSSION TITLE
Error 500 - Server got itself in trouble

### DISCUSSION THREAD

**Discussion by Suprimos85** [alandtse/alexa_media_player#2638] [General]
Hello,

even with the newest update, I still get the error 500 after the last step (solve captcha) -->server got itself in trouble

Core: 2024.10.3
Supervisor: 2024.10.2
OS: 13.2
alexa_media version: 4.13.5

**Comment by Avpman2** [comment]
"server got itself in trouble" Ditto here. Are there any plans or ability to fix this, or did Amazon cut our cord?
I've never been able to get it set up.

**Comment by danielbrunt57** [comment]
There's no issue with Amazon.  My integration is working fine.  You shouldn't see a captcha screen after your first virgin login to your Amazon account if your browser is working and saving cookies.

**Comment by kendael** [comment]
Hello! Same problem here. I tried to reinstall it from scratch but still this error 500 page.

**Comment by kendael** [comment]
Ok. I have found the problem. It seems that 2FA is now REQUIRED for this integration to work.

**Reply by Avpman2** [reply]
Do you think that means someone can/will fix it?

**Reply by danielbrunt57** [reply]
That someone will have to be every single user who is NOT using 2FA!

**Comment by danielbrunt57** [comment]
I'd like to add that the 2FA (or 2SV as Amazon calls it) has to be _ONLY_ via an authenticator application (app secret) and _NOT_ one sent via SMS.

**Reply by Bachomp** [reply]
Hmmmm, I already have 2FA enabled using OTP App (Not SMS)  and have form some years but still unable to configure the integration.
Also tried different browsers

**Comment by danielbrunt57** [comment]
There's been zero information provided like logs to analyze or detailed sequence of steps to reproduce the issue.
